README.md

```diff
@@ -1,29 +1,46 @@
 # cdist
 
-**cdist** is a usable configuration management system.
+This is [community maintained](https://github.com/cdist-community)
+fork of Ungleich's [cdist](https://github.com/ungleich/cdist)
+(after [f061fb168ddacc894cb6e9882ff5c8ba002fadd8](https://github.com/ungleich/cdist/commit/f061fb168ddacc894cb6e9882ff5c8ba002fadd8)).
 
-It adheres to the [**KISS principle**](https://en.wikipedia.org/wiki/KISS_principle)
-and is being used in small up to enterprise grade environments.
+Work is split between three repositories:
 
-For more information have a look at [**homepage**](https://cdi.st)
-or at **``docs/src``** for manual in **reStructuredText** format.
+* [cdist](https://github.com/cdist-community/cdist) - main implementation of **cdist core**.
+* [cdist-conf](https://github.com/cdist-community/cdist-conf) - **essential** explorers and types.
+* [cdist-extra](https://github.com/cdist-community/cdist-extra) - **non-essential** explorers, types, scripts, tools etc.
 
-## Contributing
+What's the difference between essential and non-essential? Explorers and types,
+which are used to explore or change state (modify files and directories,
+install packages, manage services etc) of operating system and are not stictly
+related to some specific piece of software, are considered essential.
+
+## Getting Started
 
-Pull requests can be made in both [upstream](https://code.ungleich.ch/ungleich-public/cdist)
-(managed by [**ungleich**](https://ungleich.ch)) and [GitHub](https://github.com/ungleich/cdist).
+Since this fork is still in early stages, there will be no versioned releases
+and everything is expected to be used straight from repository, because for now
+we are targeting power users of cdist, who know what they are doing.
 
-Issues can be made and other project management activites happen
-[**only in upstream**](https://code.ungleich.ch/ungleich-public/cdist)
-(needs [**ungleich** account](https://account.ungleich.ch)).
+### Get core with essentials
 
-For community-maintained types there is
-[**cdist-contrib** project](https://code.ungleich.ch/ungleich-public/cdist-contrib).
+```bash
+git clone --recurse-submodules https://github.com/cdist-community/cdist
```

Comment:
Should we have an extra step to update the conf submodule to the latest commit?

Reply:
Added.